5156 Writes: add v1 to Serverless get-started (#5233)
influxdata/docs-v2: content/influxdb/cloud-dedicated/get-started/write.md

to InfluxDB:

{{% code-placeholders "DATABASE_TOKEN" %}}

```sh

> curl --silent -w "%{http_code}\n" \

Reviewer: The [cURL write out docs](https://everything.curl.dev/usingcurl/verbose/writeout#available-write-out-variables), recommend `response_code` over `http_code`.

One concern I have with this is that it only returns the response code. While users shouldn't have any issues if the follow the tutorial, there is a chance the request could fail and more information would be helpful. Should this also output the `errormsg`? (This is left blank if there is no error).

```suggestion
curl --silent -w "%{response_code}: %{errormsg}\n" \
```

home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
"
```

{{% /code-placeholders %}}

Replace the following:

- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [token](/influxdb/cloud-dedicated/admin/tokens/) with sufficient permissions to the specified database

If successful, the output is the following HTTP status code:

<!--pytest-codeblocks:expected-output-->

```

> 204

Reviewer: You'll need to update this if you apply my suggestion above.

```suggestion
204: 
```

- **Request body**: Line protocol as plain text

{{% note %}}

> With the {{% product-name %}} v2 API `/api/v2/write` endpoint, `Authorization: Bearer` and `Authorization: Token` are equivalent and you can use either scheme to pass a database token in your request.

Reviewer: I don't know the technical differences between the `Bearer` and `Token` schemes, but I hesitate to use "equivalent" here.

```suggestion
The {{% product-name %}} v2 API `/api/v2/write` endpoint supports both `Bearer` and `Token` authorization schemes and you can use either scheme to pass a database token in your request.
```

For more information about HTTP API token schemes, see how to [authenticate API requests](/influxdb/cloud-dedicated/guides/api-compatibility/v2/).
{{% /note %}}

The following example uses cURL and the InfluxDB v2 API to write line protocol
to InfluxDB:

{{% code-placeholders "DATABASE_TOKEN"%}}
```sh

> curl --silent -w "%{http_code}\n" \

Reviewer: ```suggestion
curl --silent -w "%{response_code}: %{errormsg}\n" \
```

home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
"
```

{{% /code-placeholders %}}

Replace the following:

- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [token](/influxdb/cloud-dedicated/admin/tokens/) with sufficient permissions to the specified database

If successful, the output is the following HTTP status code:

<!--pytest-codeblocks:expected-output-->

```

> 204

Reviewer: ```suggestion
204: 
```

```

> 

Author: ```suggestion

Otherwise, the error status and message.

```
